vmngclient/utils/upgrades_helper.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+
+from attr import define  # type: ignore
+
+from vmngclient.dataclasses import Device
+from vmngclient.utils.personality import Personality
+
+
+class Family(Enum):
+    VEDGE = "vedge"
+    VMANAGE = "vmanage"
+
+
+class VersionType(Enum):
+    VMANAGE = "vmanage"
+
+
+class DeviceType(Enum):
+    CONTROLLER = "controller"
+    VEDGE = "vedge"
+    VMANAGE = "vmanage"
+
+
+@define
+class InstallSpecification:
+
+    family: Family
+    version_type: VersionType
+    device_type: DeviceType
+
+
+class InstallSpecHelper(Enum):
+
+    VMANAGE = InstallSpecification(Family.VMANAGE, VersionType.VMANAGE, DeviceType.VMANAGE)  # type: ignore
+    VSMART = InstallSpecification(Family.VEDGE, VersionType.VMANAGE, DeviceType.CONTROLLER)  # type: ignore
+    VBOND = InstallSpecification(Family.VEDGE, VersionType.VMANAGE, DeviceType.CONTROLLER)  # type: ignore
+    VEDGE = InstallSpecification(Family.VEDGE, VersionType.VMANAGE, DeviceType.VEDGE)  # type: ignore
```

Comment:
Can we attach some info about why we cannot reuse Personality?

Reply:
Sure, done.